.github/workflows/build_agent.yml

```diff
@@ -36,4 +36,5 @@ jobs:
       run: |
         source /opt/ros/${{ matrix.ros_distribution }}/setup.sh
         cd ros_ws
-        colcon build --cmake-clean-cache --cmake-force-configure --packages-up-to micro_ros_agent microxrcedds_agent --cmake-args -DMICROROSAGENT_SUPERBUILD=OFF
\ No newline at end of file
+        colcon build --cmake-clean-cache --cmake-force-configure --packages-up-to microxrcedds_agent
```

Comment:
Don't you need to make a source of `install/local_setup.sh` to add `microxrcedds_agent` to LD_LIRBARY_PATH`?

Reply:
Indeed, fixed on 757b4013d34837650955ff7b2f05deedb1de1265